tests/Unit/Product/WCProductAdapterTest.php

```diff
@@ -1577,6 +1577,35 @@ public function test_valid_image_name_with_utf8_nfd() {
 		$this->assertNull( $violation );
 	}
 
+	public function test_invalid_image_names() {
+		$product = WC_Helper_Product::create_simple_product();
+
+		$main_image = $this->generate_mock_image_attachment( $product->get_id(), '600-×-300.png' );
+
+		$additional_images = [ $this->generate_mock_image_attachment( $product->get_id(), 'æą€.jpg' ) ];
+
+		$product->set_image_id( $main_image );
+		$product->set_gallery_image_ids( $additional_images );
+		$product->save();
+
+		$adapted_product = new WCProductAdapter(
+			[
+				'wc_product'    => $product,
+				'targetCountry' => 'US',
+			]
+		);
+
+		$violation = $this->validate_product_property( $adapted_product, 'imageLink' );
+		$this->assertEquals(
+			'Product image "600-×-300.png" is not a valid name.',
```

Comment:
❓  Just curious why here it appears `Product image "600-×-300.png" is not a valid name.` and in the screenshot and code we are adding `Product image "600-×-300.png" is not a valid name. [image]`